### DISCUSSION TITLE
AWS Green Grass IPC - How to Unsubscribe from a Topic?

### DISCUSSION THREAD

**Discussion by chandar1** [aws/aws-iot-device-sdk-cpp-v2#742] [Q&A] [answered]
Hi All,

I'm able to use AWS SDK to subscribe_to_topic or subscribe_to_iotcore and receive messages.

Now I need to unsubscribe from some of the topics/iotcore - are there examples explaining how to do Unscubscription ?

Thanks,

**Comment by bretambrose** [comment, marked as answer]
Unsubscribe should happen automatically when you close a subscribe
